apps/geomat/src/app/landing/landing.component.html

```diff
@@ -30,8 +30,13 @@ <h3 matLine>
     <mat-icon svgIcon="quiz"></mat-icon>
     <span>Selbsttest</span>
   </mat-grid-tile>
+  <mat-grid-tile [routerLink]="['/quiz']" class="mat-elevation-z2">
```

Comment:
There is already an menu entry for the quiz. Please remove this duplicate.

Reply:
I removed it.